prow/plugins.yaml

```diff
@@ -346,6 +346,8 @@ plugins:
   GoogleCloudPlatform/k8s-tpu-operator:
   - approve
   - assign
+  - hold
+  - label
   - lgtm
   - size
   - trigger
```

Comment:
You need to enable the `wip` plugin here in order for the `do-not-merge/work-in-progress` label to be auto-applied to PRs with `WIP` in the title.